projects/plugins/boost/app/assets/src/js/features/performance-history/performance-history.module.scss

```diff
@@ -5,7 +5,12 @@
 		border-radius: 8px;
 	}
 
+	:global(.components-panel__body-title > button) {
+		font-size: 14px;
+		font-weight: 400;
+	}
+
```

Comment:
Any way we can just style the button directly?

Reply:
It's in another component.

Reply:
Yep, but if there are no "small" variant of that component, maybe we need to update that component instead?

Reply:
I think the problem here is that Boost uses non-Jetpack styles 😅